src/imageops/mod.rs

```diff
@@ -1,8 +1,6 @@
 //! Image Processing Functions
 use std::cmp;
 
-use num_traits::{NumCast};
```

Comment:
`NumCast` provides a `i64::from` method, which conflicts with the implementation provided by the `From` trait. I just switched the couple places that refer to NumCast to reference it by its full name.